### PR TITLE
SDT-879 Add Account header

### PR DESCRIPTION
**Pull request by rpowis**
Adding the Account Header as a layout file and styles. 

image

There needs to be some follow up work to this in order to make the language select more dynamic and to add some tests for the layout file, but that would benefit from further discussion first.

### CODE REVIEW

File: src/components/account-menu/template.njk [hunk 1]
```diff
@@ -9,7 +11,10 @@
       <a href="#" class="hmrc-account-menu__link">Back</a>
     </li>
     <li>
-      <a href="#" class="hmrc-account-menu__link">Messages<span class="hmrc-notification-badge">2</span>
+      <a href="#" class="hmrc-account-menu__link">Messages
```
Comment (nataliecarey) [inline src/components/account-menu/template.njk:14]: By introducing a line break you're introducing a space into the content - I think that's a good thing, just want to make sure you're aware of this change.

File: src/components/notification-badge/notification-badge.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 examples:
 - name: default
   data:
-    text: 3
+    badgeText: 3
 - name: with-text
   data:
-    text: New
+    badgeText: New
```
Comment (nataliecarey) [inline src/components/notification-badge/notification-badge.yaml]: Is this related to the account header? What's the reason for breaking the `text` convention govuk are using?

File: src/components/account-menu/template.njk (hunk 1 above)
Comment (rpowis) [reply src/components/account-menu/template.njk:14]: yep, that was the idea 😄 it's a much tidier way of doing it than having a space followed by the conditional all on one line

File: app/app.js [hunk 3]
```diff
@@ -12,7 +12,8 @@ const appViews = [
   configPaths.layouts,
   configPaths.views,
   configPaths.components,
-  configPaths.src
+  configPaths.src,
+  configPaths.govukFrontend + '/components'
```
Comment (nataliecarey) [inline app/app.js:16]: What problem does this solve?
